src/main/scala/Vcs.scala

```diff
@@ -106,6 +108,11 @@ class Mercurial(val baseDir: File) extends Vcs with GitLike {
 
   def currentBranch = (cmd("branch") !!) trim
 
+  // FIXME: Need help here
+  def setBranch(branch: String) = cmd("branch", branch)
+
+  def newBranch(branch: String) = setBranch(branch)
+
```

Comment:
It's been too long since I've used hg to know what the right things are here.  I don't think this is right though, I think branches get committed, so you would need to commit after calling the branch command.  As with the svn support, happy to merge if these are changed to throw an exception saying this hasn't been implemented for hg yet.